packages/x-data-grid/src/hooks/features/pagination/gridPaginationSelector.ts

```diff
@@ -16,6 +17,15 @@ const ALL_RESULTS_PAGE_VALUE = -1;
  */
 export const gridPaginationSelector = (state: GridStateCommunity) => state.pagination;
 
+/**
+ * @category Pagination
+ * @ignore - do not document.
+ */
+export const gridPaginationEnabledClientSideSelector = createSelector(
```

Comment:
This is being exported from the root `index.ts` despite being `@ignore`, it would be nice to avoid exposing it.